nengo_gui/components/spa_similarity.py

```diff
@@ -0,0 +1,102 @@
+from .pointer import Pointer
+from nengo_gui.components.component import Component
+
+import numpy as np
+import nengo
+
+import struct
+import copy
+
+class SpaSimilarity(Pointer):
```

Comment:
It seems this class overrides almost all of `Pointer`'s method. Thus, I would suggest to not derive from `Pointer`. If there is code worth to share, it might be better to create an abstract class `SpaPlot` (or another appropriate name) containing the shared code and have `Pointer` and `SpaSimilarity` derive from it.